app/__main__.py

```diff
@@ -52,18 +75,94 @@ async def cancel(self, interaction: Interaction, button: Button) -> None:
         message = await interaction.original_response()
         await interaction.edit_original_response(embed=message.embeds[0], view=self)
 
-    async def _create_embed(self, interaction: Interaction) -> discord.Embed:
+    @discord.ui.button(
+        label="CPS Upgrade 10x", style=discord.ButtonStyle.gray, row=1, custom_id="upgradepersistent:cps10x"
+    )
+    async def cps_upgrade_ten(self, interaction: Interaction, button: Button) -> None:
```

Comment:
FTR I was more thinking a button that toggles between 10x and 1x but this works too. Maybe when we have any other upgrades.